tests/testthat/_snaps/print.md

```diff
@@ -23,13 +23,12 @@
     Code
       cat(toString(matrix_form(lsting), widths = c(7, 8, 9), hsep = "-"))
     Output
+                                    
                 Descript            
-      Unique      ion               
-      Subject      Of               
-      Identif   Planned    Continous
-        ier       Arm        Level  
-                           Biomarker
-                               1    
```

Comment:
I think this is an important improvement coming from the pre-proc/compactification (if it is a word) of the string matrix before wrapping